include/dsVideoPort.h

```diff
@@ -284,8 +286,9 @@ typedef void (*dsVideoFormatUpdateCB_t)(dsHDRStandard_t videoFormat);
 /**
  * @brief Checks whether a video port is active or not.
  * 
- * This function is used to indicate whether a video port is active or not. A HDMI output port is active if it is connected 
- * to the active port of sink device. 
+ * For Source devices, this function is used to indicate whether a video port is active or not. A HDMI output port is active if it is connected
+ * to the active port of sink device.
+ * For sink devices, this function returns the active status as true always. Please refer :: dsVIDEOPORT_TYPE_INTERNAL (Sink's own display)
```

Comment:
It would be better if we add a bit more info as, "For Sink devices, where the display is always connected (dsVIDEOPORT_TYPE_INTERNAL), this API will return true for the Video port status."

Reply:
Updated